virttest/libvirt_cgroup.py

```diff
@@ -154,10 +156,12 @@ def __get_cg_file_path(cg_key, cg_path, cg_file_name):
             dev_init_dict = {"rbps": "max", "wbps": "max", "riops": "max",
                              "wiops": "max"}
             dev_list = []
+            if not self.is_bfq():
+                CGROUP_V1_BLKIO_FILE_MAPPING['weight'] = 'blkio.weight'
             for cg_key, cg_file_name in list(CGROUP_V1_BLKIO_FILE_MAPPING.items()):
                 cg_file_path = __get_cg_file_path(cg_key, cgroup_path, cg_file_name)
                 with open(cg_file_path, 'r') as cg_file:
-                    if cg_key in ["weight"]:
+                    if cg_key in ["weight", "weight_device"]:
```

Comment:
the weight_device is like other device specific param such as 'device_read_iops_sec', so I guess it should be put in the "if cg_key in ["rbps", "wbps", "riops", "wiops"]:" branch.
And the device recorded as the key in the standardized dict in the format of dev's major:minor number. 
for example, our current test info as follow:
`2021-04-22 03:55:25,997 control_cgroup   L0229 DEBUG| blkiotune: The input info is: {'252:0': {'rbps': '345600000', 'wbps': 'max', 'riops': 'max', 'wiops': 'max'}}The ouput info is: {'weight': '100', '252:0': {'rbps': '345600000', 'wbps': 'max', 'riops': 'max', 'wiops': 'max'}}The cgroup info is:{'weight': '100', '252:0': {'rbps': '345600000', 'wbps': 'max', 'riops': 'max', 'wiops': 'max'}}`
the 252:0 is the device such as the /dev/sda